Source/WebKit/NetworkProcess/cocoa/NetworkSessionCocoa.mm

```diff
@@ -1949,9 +1944,8 @@ - (void)URLSession:(NSURLSession *)session webSocketTask:(NSURLSessionWebSocketT
     auto addResult2 = m_perPageSessionSets.add(pageID, SessionSet::create());
     ASSERT(addResult2.isNewEntry);
     RetainPtr<NSURLSessionConfiguration> configuration = adoptNS([m_defaultSessionSet->sessionWithCredentialStorage.session.get().configuration copy]);
-#if HAVE(CFNETWORK_NSURLSESSION_ATTRIBUTED_BUNDLE_IDENTIFIER) && USE(APPLE_INTERNAL_SDK)
```

Comment:
Do we still need `CFNETWORK_NSURLSESSION_ATTRIBUTED_BUNDLE_IDENTIFIER` in `PlatformHave.h`?

Reply:
I don't know, honestly.

Reply:
> Do we still need `CFNETWORK_NSURLSESSION_ATTRIBUTED_BUNDLE_IDENTIFIER` in `PlatformHave.h`?

I removed it.